color-changer.js

```diff
@@ -0,0 +1,42 @@
+function getRandomUnusedHue(usedHues) {
```

Comment:
Sure, I can wrap it all up in an IIFE. If you'd like to use it in some other way than just copy/paste into chrome, I could have that IIFE return a function call that initiates the random background change, but I'll keep to the spirit of copy/paste into chrome for my upcoming changes.